net/miniupnpd/files/upnpd.config

```diff
@@ -2,6 +2,7 @@ config upnpd config
 	option enabled		0
 	option enable_natpmp	1
 	option enable_upnp	1
+	option force_forwarding 1
```

Comment:
Please don't make this option default. Previously when you made `ext_ip_reserved_ignore` default, it resulted in cryptic syslog messages for miniupnpd users outside of China.